app/v2/controllers/V2ArrivalsController.scala

```diff
@@ -132,6 +137,20 @@ class V2ArrivalsControllerImpl @Inject() (
         } yield arrivalResult
     }
 
+  def getArrivalsForEori(updatedSince: Option[OffsetDateTime]): Action[AnyContent] =
```

Comment:
can not see any usage of updatedSince:?

Reply:
Interstingly, this isn't on the equivalent method on the Departures controller either, so I wonder if it's supported in the backend (transit-movements). We should probably look to fix that via another ticket if it isn't. If it is supported in the backend, then it should be supported here, and a ticket should be raised for the departures side of things.

Reply:
Yes, by default we sort by descending order in _transit-movements_     
```
val aggregates = Seq(
      Aggregates.filter(selector),
      Aggregates.sort(descending("updated")),
      Aggregates.project(projection)
    )
```